### ISSUE TITLE
Potential imagery sources for German states

### ISSUE THREAD

**Issue by arch0345** [osmlab/editor-layer-index#2443]
We currently have imagery layers for 8 out of the 16 states in Germany. The imagery for the remaining 8 states are listed below. These are either licensed under DL-DE->BY-2.0 or CC BY 4.0, so explicit permission needs to be granted from these agencies before they are added to ELI.

- [x] Baden-Württemberg: https://www.geoportal-bw.de/geonetwork/srv/ger/catalog.search#/metadata/4927a198-e9ce-da9a-a5ef-c11f21ff173f (#2737)
- [ ] Bremen: https://metaver.de/trefferanzeige?docuuid=3dd11c0f-9c13-4ada-8b3a-885aa979e329
- [x] Mecklenburg-Vorpommern: https://advmis.geodatenzentrum.de/trefferanzeige?docuuid=b6a74a2b-9d69-4f74-ba53-21111da7851f (#2816)
- [x] Niedersachsen: https://ni-lgln-opengeodata.hub.arcgis.com/apps/lgln-opengeodata::digitales-orthophoto-dop20/about (#2807)
- [x] Rheinland-Pfalz: https://www.geoportal.rlp.de/mapbender/php/mod_showMetadata.php?languageCode=en&resource=layer&layout=tabs&id=61676#tabs-5 (#2806)
- [x] Saarland: https://geoportal.saarland.de/mapbender/php/mod_showMetadata.php?languageCode=en&resource=layer&layout=tabs&id=46303#tabs-5 (#2507)
- [ ] Schleswig-Holstein: https://opendata.schleswig-holstein.de/dataset/digitale-orthofotos-1-5000dop20
- [x] Thüringen: https://inspire-geoportal.ec.europa.eu/srv/api/records/10cbd9e4-8a84-4771-8ed1-a3cf66a01766?language=all (#2495)

**Comment by arch0345** [comment]
@coolultra1 would you be able to contact these government agencies? I would, but I figure it would be better for someone fluent in German to do so

**Comment by coolultra1** [comment]
@arch0345 Of course, thanks for collecting these. It could be German OSM contributors already tried to contact some, so I'll check the wiki first.

Will do this within this week.

**Comment by coolultra1** [comment]
Thuringia says yes - Will add soon.

**Comment by rbuffat** [comment]
BW seems to have an Open Data (Datenlizenz Deutschland - Namensnennung - Version 2.0 https://www.govdata.de/dl-de/by-2-0.) digital terrain model in 0.25m resolution which would be great to map paths in forests: https://opengeodata.lgl-bw.de/#/(sidenav:product/dgm025). But unsure if they offer it as an image service and I guess it's the same issue regarding the attribution.
